tests/test_subannual.R

```diff
@@ -0,0 +1,293 @@
+library(testthat)
+library(tsbox)
+
+t3 <- ts_rbind(ts_dt(mdeaths), AirPassengers)   ##### this function contains error, should have both
+# AirPassengers and mdeaths in column "var"
+t4 <- ts_rbind(ts_xts(AirPassengers), ts_tbl(mdeaths)) ##### this function contains error, should have both
+# AirPassengers and mdeaths in column "var"
+
+
+context("irregular conversion handling")
+
+####### Monthly series: AirPassengers #####
```

Comment:
Rather than having comments here, this should be in the `test_that` string. We should try to be as explicit as we can in the string or in the context, as this text is shown if a test fails. 

Have a look at the dplyr to see how tests should look like: https://github.com/tidyverse/dplyr/blob/master/tests/testthat/test-arrange.r